timemachine/cpp/src/bd_exchange_move.cu

```diff
@@ -162,15 +165,18 @@ void BDExchangeMove<RealType>::move(
     curandErrchk(templateCurandUniform(cr_rng_translations_, d_translations_.data, d_translations_.length));
     curandErrchk(templateCurandUniform(cr_rng_samples_, d_sample_noise_.data, d_sample_noise_.length));
     curandErrchk(templateCurandUniform(cr_rng_mh_, d_mh_noise_.data, d_mh_noise_.length));
-    for (int step = 0; step < steps_per_move_; step++) {
+    size_t step = 0;
+    // For the first pass just set the value to zero on the host
+    p_noise_offset_.data[0] = 0;
```

Comment:
style-nit: should this be just `*p_noise_offset_.data = 0` since it's a pointer to a scalar?

Reply:
Switched dereference style in https://github.com/proteneer/timemachine/pull/1274/commits/9a1895a617de3ccf6adb544aa785035bf1a4568a https://github.com/proteneer/timemachine/pull/1274/commits/9a1895a617de3ccf6adb544aa785035bf1a4568a